roles/osg/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Create osg role
+  os_keystone_role:
+    auth: "{{ openstack_auth }}"
+    name: osg
+    
+- name: "Add service user to osg project as osg"
+  os_user_role:
+    auth: "{{ openstack_auth }}"
+    user: "{{ osg_service_username }}"
+    role: osg
+    project: "{{ osg_project_id }}"
+    
+- name: Pull Docker image
+  docker_image:
+    source: pull
+    name: "{{ osg_deployer_docker_image }}"
+    force_source: yes
+
+- name: Create config directory
+  file:
+    path: "{{ osg_config_path }}"
+    state: directory
+
+- name: Create osg deployer conf
+  template:
+    src: "osg.conf.j2"
+    mode: 0640
+    dest: "{{ osg_config_path }}/osg.conf"
+
+- name: Create osg backfill container env file
+  template:
+    src: "osg.env.j2"
+    mode: 0640
+    dest: "{{ osg_config_path }}/osg.env"
+    
+- name: Create openstack env file
+  template:
+    src: "openstack.env.j2"
+    mode: 0640
+    dest: "{{ osg_config_path }}/openstack.env"
+
+- name: Configure osg deployer docker bash wrapper
+  template:
+    src: osg-deployer.j2
+    mode: a+x
```

Comment:
I'm not sure, but I thought things in /sbin don't have executable by all set. It would be more like 0750. But maybe double-check that.